ardueshop/base/templates/base/base.html

```diff
@@ -52,7 +52,7 @@
           {% if user.is_authenticated %}
             <div class="d-flex align-items-center">
               <p class="nav-link m-0 me-3">
-                <strong>{{ user.username }}</strong>
+                <strong>Tu</strong>
```

Comment:
It works fine, but why have you removed the username from the navbar? I think it's better if the navbar shows your username/email instead of "Tu".